third_party/amd/lib/TritonAMDGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -1495,14 +1495,7 @@ static SmallVector<Value> S8_to_Bf16(Location loc,
   SmallVector<Value> outValues = {};
   for (Value inVal : inValues) {
     Value i32Val = b.sext(i32_ty, inVal);
-
-    GCNBuilder builder;
-    auto &cvt = *builder.create("v_cvt_f32_i32");
-    auto res = builder.newOperand("=v");
-    auto operand = builder.newOperand(i32Val, "v");
-    cvt(res, operand);
-    auto f32Val = builder.launch(rewriter, loc, f32_ty, false);
-
+    Value f32Val = rewriter.create<LLVM::SIToFPOp>(loc, f32_ty, i32Val);
```

Comment:
We don't need to have the explicit `sext` in the previous line? Actually I'm wondering whether we can directly use `SIToFPOp` to convert from s8 to bf16 without needing the manual shift following this too.